CONTRIBUTING.md

```diff
@@ -30,6 +30,8 @@ License Agreement before we can accept your pull request: https://cla.puppet.com
 
 ## Installing Bolt
 
+If you are interested in trying bolt out or using in production we recommend installing from a system package detailed in [installing bolt](https://puppet.com/docs/bolt/latest/bolt_installing.html). The following installation instructions are focused on developers who wish to contribute to Bolt.
+
 Depending on your development workflow, you can install Bolt one of three ways:
 
 * From [RubyGems](https://rubygems.org)
```

Comment:
Should we include `bundle exec rake install` and `install:local`? This seems like it's more about building the gem and installing than just installing the gem, and those both play more nicely with ruby environments and won't confuse users about installing bolt for 'real use' on their system with packages.

Reply:
I dont really understand how to document your suggestion. I have used a workflow like `gem build bolt` and then `gem install bolt-[VERSION].gem`  are those rake tasks `install` and `install::local` built ins? They don't appear to be defined in Rakefile. Is that in our repo somewhere?

Reply:
Are they not there for you? If I do `bundle exec rake -T` from the root of bolt I get:
```
...
rake install                      # Build and install bolt-1.13.0.gem int...
rake install:local                # Build and install bolt-1.13.0.gem int...
...
```
Those tasks will build the gem and either install to the system gem environment, or the local env (so `bundle exec bolt` will be the built version).

Reply:
So they appear to be part of bundler https://github.com/bundler/bundler/blob/bca8a4bada7bbedab36bb04d6bdce19630d1b873/lib/bundler/gem_helper.rb#L50-L53
Will update that section. Thanks.